<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/civiltools/images/automatic.svg</normaloff>:/civiltools/images/automatic.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>استان</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Gui::PrefComboBox" name="city">
            <property name="prefEntry" stdset="0">
             <cstring>city</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_4">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>شهر</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_5">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>نوع خاک</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="Gui::PrefComboBox" name="soil_type">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>soiltype</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>I</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>II</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>III</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IV</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="Gui::PrefComboBox" name="ostan">
            <property name="prefEntry" stdset="0">
             <cstring>ostan</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_14">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>خطر نسبی</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="Gui::PrefComboBox" name="risk_level">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>soiltype</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>خیلی زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>متوسط</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>کم</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="abir_groupbox">
          <property name="title">
           <string>A*B*I / R</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="Gui::PrefComboBox" name="importance_factor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <property name="prefEntry" stdset="0">
              <cstring>importance_factor</cstring>
             </property>
             <property name="prefPath" stdset="0">
              <cstring>Mod/civilTools</cstring>
             </property>
             <item>
              <property name="text">
               <string>0.8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.4</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="rux_label">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Rux</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QDoubleSpinBox" name="ruy">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>11.000000000000000</double>
             </property>
             <property name="value">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QDoubleSpinBox" name="rux">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>1</number>
             </property>
             <property name="minimum">
              <double>2.000000000000000</double>
             </property>
             <property name="maximum">
              <double>11.000000000000000</double>
             </property>
             <property name="value">
              <double>5.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="importance_factor_label">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>ضریب اهمیت</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="ruy_label">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Ruy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="create_pushbutton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/spectral.svg</normaloff>:/civiltools/images/spectral.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_pushbutton">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="../../civiltools.qrc">
           <normaloff>:/civiltools/images/cancel.svg</normaloff>:/civiltools/images/cancel.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>X System</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeView" name="x_treeview">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Y System</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTreeView" name="y_treeview">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="spectral"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ostan</tabstop>
  <tabstop>city</tabstop>
  <tabstop>risk_level</tabstop>
  <tabstop>soil_type</tabstop>
  <tabstop>y_treeview</tabstop>
  <tabstop>create_pushbutton</tabstop>
  <tabstop>cancel_pushbutton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
